webpack-babel.config.js

```diff
@@ -0,0 +1,20 @@
+var path = require('path');
+
+module.exports = {
+  output: {
+    libraryTarget: 'umd',
```

Comment:
Btw, are we sure the target is nodejs? ppl will `import` or `require` our sources from a browser right? Or maybe I misunderstood something?